src/lib/es5.d.ts

```diff
@@ -594,6 +594,20 @@ interface TemplateStringsArray extends ReadonlyArray<string> {
 interface ImportMeta {
 }
 
+/**
+ * The type of `import` in `import.meta`.
+ */
+interface ImportMetaExpression {
+    meta: ImportMeta;
+}
+
+/**
+ * The type of `new` in `new.target`.
+ */
+interface NewTargetExpression<T extends NewableFunction> {
+    target: T;
+}

```

Comment:
I'm not sure about adding these. Is this the way to do it or is there a better way?

Reply:
I think you could do it fully synthetically, but this is probably easiest and seems analogous to how `arguments.callee` works.

Reply:
Cool! Is the name ok? `ImportMetaKeyword`/`NewTargetKeyword`? I'm not sure.

Reply:
Hm yeah, these are tricky. I don’t have a better suggestion than what you have now.

Reply:
I'm not sure we should have augmentable interfaces for these. Consider that if you add `foobar` to one and wrote `new.foobar` (as completions prompts you), I believe we'd still issue an error on it because it's not a valid psuedoproperty. So as to not mislead people on the capability of these structures, we might want to keep the type wholly synthetic.

In any case, I think the properties should also be marked `readonly`, too (synthetic or no).

Reply:
@weswigham is there a good example somewhere that I could copy for making these synthetic?

Reply:
Ok, I might have figured this out in e328f85b21b03e6658721fae75c0aa7c76e7280e then made the properties readonly in 8eac25218b5453600128078585ba0c5559aa277c